src/main/java/com/uid2/admin/vertx/service/AdminKeyService.java

```diff
@@ -374,7 +396,7 @@ private List<OperationModel> handleAdminRoles(RoutingContext rc) {
      * @param a the AdminUser to write
      * @return a JsonObject representing a, without the key field.
      */
-    public JsonObject adminToJson(AdminUser a){
+    public JsonObject adminToJson(AdminUser a){ // should hash key
```

Comment:
If you mean to hash the key then do it? Don't just leave a comment to do it.

Reply:
woops that slipped, thanks